From 53f9d2bf282de1300c194184bc9dc39201cbecee Mon Sep 17 00:00:00 2001
From: Bugdebugger <bugdebugger@users.noreply.github.com>
Date: Wed, 8 Apr 2015 21:09:35 +0200
Subject: [PATCH 106/129] Update shown properties in editor for moved object

The object property values shown in the editor now are in sync with the
actual values when an object is being dragged around with the mouse
---
 src/editor/viewport.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/editor/viewport.cpp b/src/editor/viewport.cpp
index bca02b776..fb8c864bd 100644
--- a/src/editor/viewport.cpp
+++ b/src/editor/viewport.cpp
@@ -198,6 +198,7 @@ Viewport::on_primary_button_release(int x_, int y_)
     {
       (*it)->set_orig_pos((*it)->get_pos());
     }
+    selection_changed(selection);
   }
   current_action = NOTHING;
 }
@@ -238,6 +239,7 @@ Viewport::on_pointer_move(int x_, int y_)
           }
           (*it)->set_pos(Vector3f(new_x, new_y, orig_pos.z));
         }
+        selection_changed(selection);
       }
       break;
 
@@ -788,6 +790,7 @@ Viewport::move_objects(const Vector2i& offset)
                             p.y + static_cast<float>(offset.y),
                             p.z));
   }
+  selection_changed(selection);
 }
 
 Vector2i
-- 
2.51.2

